spec/ar_query_matchers/mock_data_model.rb

```diff
@@ -22,7 +22,7 @@ def self.name
         'MockUser'
       end
 
-      has_many :mock_posts
+      has_many :mock_posts, dependent: :destroy
```

Comment:
Minor modification which only hits whenever deletions occur. Granted in a real app this is likely a bad idea, but for tests it seemed reasonable.